kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -210,13 +210,17 @@ def load_manifest(yaml_contents: str) -> Iterator[HelmTemplate]:
     def __execute(self, command: list[str]) -> str:
         command = self.__set_global_flags(command)
         log.debug(f"Executing {' '.join(command)}")
-        process = subprocess.run(
-            command,
-            check=True,
-            capture_output=True,
-            text=True,
-        )
-        Helm.parse_helm_command_stderr_output(process.stderr)
+        try:
+            process = subprocess.run(
+                command,
+                check=True,
+                capture_output=True,
+                text=True,
+            )
+        except subprocess.CalledProcessError as e:
+            Helm.parse_helm_command_stderr_output(e.stderr)
+            log.debug(e.stdout)
```

Comment:
Isn't the log duplicated now? And what's the advantage of the new behavior compared with `check=False` prior?

Reply:
There is no difference in the behavior. The code is a bit more explicit, but I don't see a clear advantage over `check=False`. Should I change it to that? It's gonna get rid of some code at least.

Reply:
Not duplicated because of the `return`